test/e2e/common_test.go

```diff
@@ -844,9 +847,27 @@ func populateCache(podman *PodmanTestIntegration) {
 	fmt.Printf("-----------------------------\n")
 }
 
+func cachePath(path string) string {
+	cacheDir := os.Getenv("TMPDIR")
+	if cacheDir == "" {
+		cacheDir = "/tmp"
+	}
+
+	if path != "" {
+		cacheDir = filepath.Join(cacheDir, path)
+	}
+
+	return cacheDir
+}
+
 func removeCache() {
 	// Remove cache dirs
-	if err := os.RemoveAll(ImageCacheDir); err != nil {
+	imageCacheDir := cachePath("imagecachedir")
+	if os.Getenv("PODMAN_TEST_IMAGE_CACHE_DIR") != "" {
+		imageCacheDir = os.Getenv("PODMAN_TEST_IMAGE_CACHE_DIR")
+	}
+
+	if err := os.RemoveAll(imageCacheDir); err != nil {
```

Comment:
I'm not too happy with the possibility of someone (inadvertently or not) setting `PODMAN_TEST_IMAGE_CACHE_DIR=/tmp` (or `/etc` or even `/`) and having that be wiped

Reply:
I changed a condition for remove a cache directory.